package.json

```diff
@@ -39,6 +39,7 @@
   "devDependencies": {
     "gh-pages": "~0.3.1",
     "moment": "~2.3.0",
+    "shallow-settings": "^0.1.0",
```

Comment:
Another fun protip, setting `save` via `npm config` guarantees a `--save` is always run in case you forget:

```bash
npm config set save true
```